libs/pages/onboarding/ui/src/lib/components/step-pricing/step-pricing.tsx

```diff
@@ -1,10 +1,179 @@
+import { Button, ButtonSize, ButtonType, InputSelect, Value } from "@console/shared-ui"
+import { ONBOARDING_URL, ONBOARDING_MORE_URL } from "@console/shared/utils"
+import { InputRadio } from "@console/shared-ui"
+import { useState } from "react"
+
 /* eslint-disable-next-line */
 export interface StepPricingProps {}
 
 export function StepPricing(props: StepPricingProps) {
+
+  const [select, setSelect] = useState('free');
+  const [pro, setPro] = useState('300');
+  const [business, setBusiness] = useState('1000');
+  const [defaultValue, SetDefaultValue] = useState({
+    pro: {
+      label: '300/month',
+      value: '300'
+    },
+    business: {
+      label: '1000/month',
+      value: '1000'
+    }
+  })
+
+  const planChoose = select;
+  const displayDeploy = planChoose === 'pro' || planChoose === 'business';
+
+  const plans = [
```

Comment:
Need to be on an external file like `pricing.enum.ts` with all your constants it's very huge here